refinery/data_set_manager/tasks.py

```diff
@@ -233,7 +231,8 @@ def create_dataset(investigation_uuid, username, identifier=None, title=None,
 
     datasets = DataSet.objects.filter(name=dataset_name)
     # check if the investigation already exists
-    if len(datasets):  # if not 0, update dataset with new investigation
+    # if not 0, update dataset with new investigation
+    if len(datasets):
```

Comment:
Why do I need to check for `and not existing_dataset_uuid` below in an identical `if`?